### DISCUSSION TITLE
Mobile support?

### DISCUSSION THREAD

**Discussion by pianoman723** [coddingtonbear/obsidian-csv-table#14] [General]
Hey there, just wondering if you're planning to support Mobile? Both of my Android devices report "Failed to load plugin obsidian-csv-table" whe I attempt to enable.  Thanks!

**Comment by coddingtonbear** [comment]
Unfortunately, mobile support is not possible given that the library we're using for loading CSV files relies on Node.JS APIs that are not available in Obsidian mobile.
